app/components/primer/label_component.rb

```diff
@@ -40,21 +45,27 @@ class LabelComponent < Primer::Component
     #   <%= render(Primer::LabelComponent.new(scheme: :warning)) { "Warning" } %>
     #   <%= render(Primer::LabelComponent.new(scheme: :danger)) { "Danger" } %>
     #
+    # @example Sizes
+    #   <%= render(Primer::LabelComponent.new) { "Medium" } %>
+    #   <%= render(Primer::LabelComponent.new(size: :large)) { "Large" } %>
+    #
     # @example Variants
     #   <%= render(Primer::LabelComponent.new) { "Default" } %>
-    #   <%= render(Primer::LabelComponent.new(variant: :large)) { "Large" } %>
+    #   <%= render(Primer::LabelComponent.new(variant: :inline)) { "Inline" } %>
     #
     # @param tag [Symbol] <%= one_of(Primer::LabelComponent::TAG_OPTIONS) %>
     # @param scheme [Symbol] <%= one_of(Primer::LabelComponent::SCHEME_MAPPINGS.keys) %>
+    # @param size [Symbol] <%= one_of(Primer::LabelComponent::SIZE_OPTIONS) %>
     # @param variant [Symbol] <%= one_of(Primer::LabelComponent::VARIANT_OPTIONS) %>
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
-    def initialize(tag: DEFAULT_TAG, scheme: nil, variant: nil, **system_arguments)
+    def initialize(tag: DEFAULT_TAG, scheme: nil, size: :medium, variant: nil, **system_arguments)
```

Comment:
do we know how many uses of `variant: :large` exist in dotcom? Depending on the number we may want to write a linter to migrate them for us